test/test_numerizer_en.rb

```diff
@@ -127,4 +139,48 @@ def test_ordinal_strings
     end
   end
 
-end
+  def test_ambiguous_cases
+    # Quarter ( Coin ) is Untested
+    # Second ( Time / Verb ) is Untested
+    assert_equal 'the 4th', Numerizer.numerize('the fourth')
+    assert_equal '1/3 of', Numerizer.numerize('a third of')
+    assert_equal '4th', Numerizer.numerize('fourth')
+    assert_equal '2nd', Numerizer.numerize('second')
+    assert_equal 'I quarter', Numerizer.numerize('I quarter')
+    assert_equal 'You quarter', Numerizer.numerize('You quarter')
+    assert_equal 'I want to quarter', Numerizer.numerize('I want to quarter')
+    assert_equal 'the 1st 1/4', Numerizer.numerize('the first quarter')
+    assert_equal '1/4 pound of beef', Numerizer.numerize('quarter pound of beef')
+    assert_equal 'the 2nd second', Numerizer.numerize('the second second')
+    assert_equal 'the 4th second', Numerizer.numerize('the fourth second')
+    assert_equal '1 second', Numerizer.numerize('one second')
+
+  # TODO: Find way to distinguish this verb
+  # assert_equal 'I peel and quarter bananas', Numerizer.numerize('I peel and quarter bananas')
+  end
+
+  def test_ignore
+    assert_equal 'the second day of march', Numerizer.numerize('the second day of march', ignore: ['second'])
+    assert_equal 'quarter', Numerizer.numerize('quarter', ignore: ['quarter'])
+    assert_equal 'the five guys', Numerizer.numerize('the five guys', ignore: ['five'])
+    assert_equal 'the fifty 2', Numerizer.numerize('the fifty two', ignore: ['fifty'])
+  end
+
+  def test_bias_ordinal
+    assert_equal '4th', Numerizer.numerize('fourth', bias: :ordinal)
+    assert_equal '12th', Numerizer.numerize('twelfth', bias: :ordinal)
+    assert_equal '2nd', Numerizer.numerize('second', bias: :ordinal)
+    assert_equal '3/5', Numerizer.numerize('three fifths', bias: :ordinal)
+    assert_equal '1 4th of', Numerizer.numerize('a fourth of', bias: :ordinal)
+    assert_equal 'the 1st 2nd 3rd',  Numerizer.numerize('the first second third', bias: :ordinal)
+  end
+
+  def test_bias_fractional
+    assert_equal '1/4', Numerizer.numerize('fourth', bias: :fractional)
+    assert_equal '1/12', Numerizer.numerize('twelfth', bias: :fractional)
+    assert_equal 'the 1/4', Numerizer.numerize('the fourth', bias: :fractional)
+    assert_equal '2.75', Numerizer.numerize('two and three fourths', bias: :fractional)
```

Comment:
It would probably be good to have the same cases between ordinal and fractional bias so you can easily see the differences